sdk/containerorchestratorruntime/Azure.ResourceManager.ContainerOrchestratorRuntime/src/Generated/Models/StorageClassTypePropertiesUpdate.cs

```diff
@@ -0,0 +1,109 @@
+        /// <param name="onDelete"> The action to take when a NFS volume is deleted. Default is Delete. </param>
+        /// <param name="source"> SMB Source. </param>
+        /// <param name="username"> Server username. </param>
+        /// <param name="password"> Server password. </param>
+        /// <param name="domain"> Server domain. </param>
+        /// <param name="serializedAdditionalRawData"> Keeps track of any properties unknown to the library. </param>
+        internal StorageClassTypePropertiesUpdate(string backingStorageClassName, string azureStorageAccountName, string azureStorageAccountKey, string server, string share, string subDir, string mountPermissions, NfsDirectoryActionOnVolumeDeletion? onDelete, string source, string username, string password, string domain, IDictionary<string, BinaryData> serializedAdditionalRawData)
+        {
+            BackingStorageClassName = backingStorageClassName;
+            AzureStorageAccountName = azureStorageAccountName;
+            AzureStorageAccountKey = azureStorageAccountKey;
+            Server = server;
+            Share = share;
+            SubDir = subDir;
+            MountPermissions = mountPermissions;
+            OnDelete = onDelete;
+            Source = source;
+            Username = username;
+            Password = password;
+            Domain = domain;
+            _serializedAdditionalRawData = serializedAdditionalRawData;
+        }
+
+        /// <summary> The backing storageclass used to create new storageclass. </summary>
+        public string BackingStorageClassName { get; set; }
+        /// <summary> Azure Storage Account Name. </summary>
+        public string AzureStorageAccountName { get; set; }
+        /// <summary> Azure Storage Account Key. </summary>
+        public string AzureStorageAccountKey { get; set; }
+        /// <summary> NFS Server. </summary>
+        public string Server { get; set; }
+        /// <summary> NFS share. </summary>
+        public string Share { get; set; }
+        /// <summary> Sub directory under share. If the sub directory doesn't exist, driver will create it. </summary>
+        public string SubDir { get; set; }
+        /// <summary> Mounted folder permissions. Default is 0. If set as non-zero, driver will perform `chmod` after mount. </summary>
+        public string MountPermissions { get; set; }
+        /// <summary> The action to take when a NFS volume is deleted. Default is Delete. </summary>
+        public NfsDirectoryActionOnVolumeDeletion? OnDelete { get; set; }
+        /// <summary> SMB Source. </summary>
+        public string Source { get; set; }
+        /// <summary> Server username. </summary>
+        public string Username { get; set; }
+        /// <summary> Server password. </summary>
+        public string Password { get; set; }
+        /// <summary> Server domain. </summary>
+        public string Domain { get; set; }
```

Comment:
Is this a `Uri`?

Reply:
No. It can be an anything used to access a server, including IP